proto/ocpaths.proto

```diff
@@ -26,6 +26,19 @@ package openconfig.ocpaths;
 
 option go_package = "github.com/openconfig/featureprofiles/proto/ocpaths_go_proto;ocpaths";
 
+// Release is at minimum a timestamp for associated with an artifact.
+// By convention, itt is expected to uniquely identify and artifact
+message Release {
+  // the date/time of the relase
+  google.protobuf.Timestamp date = 1;
+
+  // the optional version of the release, typically numbers and dots.
+  optional string version = 2;
```

Comment:
in proto3 syntax there's no need to use `optional` syntax unless you need an explicit presence indicator for the field (any field can be omitted)

Reply:
Usually, comments (or a separate README/API contract document) are used to indicate mandatory fields when there's a need. I think you already did that in the parent comment (above the `message Release`). I don't think there's a way to enforce it in the syntax.

Although I wouldn't necessarily agree that specifying a release date without a release version is a great idea :)

>and mutual exclusion.

this can be done via `oneof`

Reply:
Use a comment above the field to indicate mandatory